spaghetto/src/Spago/Config.purs

```diff
@@ -322,7 +325,10 @@ data WithTestGlobs
   | OnlyTestGlobs
 
 sourceGlob :: WithTestGlobs -> PackageName -> Package -> Array String
-sourceGlob withTestGlobs name package = map (\p -> Path.concat [ getPackageLocation name package, p ])
+sourceGlob withTestGlobs name package = map snd $ sourceGlob' withTestGlobs name package
+
+sourceGlob' :: WithTestGlobs -> PackageName -> Package -> Array (Tuple String String)
```

Comment:
Do we need this change at all? Every `WorkspacePackage` contains a `path`, so we could figure the source location from there:
https://github.com/purescript/spago/blob/b92239d18971779966c15460e03166acbc9c042a/spaghetto/src/Spago/Config.purs#L109-L110